ruff.toml

```diff
@@ -0,0 +1,19 @@
+target-version = "py310"
+
+ignore = ["ANN101", "ANN002", "ANN003", "ANN401"]
```

Comment:
An interesting thing about 401:
* Apparently it is [disabled by default](https://pypi.org/project/flake8-annotations/) (at least by flake8-annotations, maybe not when used via ruff)
* [Ruff has an option](https://docs.astral.sh/ruff/settings/#flake8-annotations-allow-star-arg-any) we could use to specify that `Any` is not allowed as an explicit type, but can still be used for `*args` and `**kwargs` types

Not sure if that is what we want but thought I'd raise it!